intake/services/events_service.py

```diff
@@ -1,29 +1,169 @@
+    log_to_mixpanel(
+        distinct_id=visitor.get_uuid(),
+        event_name=event_name,
+        url=url,
+        referrer=visitor.referrer,
+        source=visitor.source)
+
+
+def site_entered(visitor, url):
+    event_name = 'site_entered'
+    log_to_mixpanel(
+        distinct_id=visitor.get_uuid(),
+        event_name=event_name,
+        url=url,
+        referrer=visitor.referrer,
+        source=visitor.source)
+
+
+def app_transferred(old_application, new_application, user):
+    event_name = 'app_transferred'
+    log_to_mixpanel(
+        distinct_id=old_application.form_submission.get_uuid(),
+        event_name=event_name,
+        user_email=user.email,
+        from_application_id=old_application.id,
+        to_application_id=new_application.id,
+        from_organization_name=old_application.organization.name,
+        to_organization_name=new_application.organization.name)
+
+
+def tags_added(tag_links):
+    event_name = 'app_tag_added'
+    for tag_link in tag_links:
+        log_to_mixpanel(
+            distinct_id=tag_link.content_object.get_uuid(),
+            event_name=event_name,
+            tag_name=tag_link.tag.name,
+            author_email=tag_link.user.email)
+
+
+def note_added(submission, user):
+    event_name = 'app_note_added'
+    log_to_mixpanel(
+        distinct_id=submission.get_uuid(),
+        event_name=event_name,
+        author_email=user.email)
+
+
+def followup_sent(submission, contact_methods):
+    event_name = 'app_followup_sent'
+    log_to_mixpanel(
+        distinct_id=submission.get_uuid(),
+        event_name=event_name,
+        contact_info_types=contact_methods)
+
+
+def confirmation_sent(submission, contact_methods):
+    event_name = 'app_confirmation_sent'
+    log_to_mixpanel(
+        distinct_id=submission.get_uuid(),
+        event_name=event_name,
+        contact_info_types=contact_methods)
+
+
+def apps_opened(applications, user):
+    event_name = 'app_opened'
+    for application in applications:
+        log_to_mixpanel(
+            distinct_id=application.form_submission.get_uuid(),
+            event_name=event_name,
+            application_id=application.id,
+            application_organization_name=application.organization.name,
+            user_email=user.email,
+            user_organization_name=user.profile.organization.name)
+
+
+def bundle_opened(bundle, user):
+    event_name = 'app_bundle_opened'
+    for submission in bundle.submissions.all():
+        log_to_mixpanel(
+            distinct_id=submission.get_uuid(),
+            event_name=event_name,
+            bundle_id=bundle.id,
+            bundle_organization_name=bundle.organization.name,
+            user_email=user.email,
+            user_organization_name=user.profile.organization.name)
 
 
-def log_form_validation_errors(request, errors):
-    return models.ApplicationEvent.log_app_errors(
-        applicant_id=get_app_id(request), errors=errors)
+def status_updated(status_update):
```

Comment:
Add new fields pruned from StatusNotificationAnalysisSerializer (for addl info diffs, etc.)